### PR TITLE
Add mobile navbar

### PR DESCRIPTION
**Pull request by alshakerM**
Added a different NavBar for mobile users.
Changed margin from pixel to percentage for responsive reasons.

### CODE REVIEW

**Review by alshakero — COMMENTED** [review]
Wow, great work. I have some notes but I'm really toasted to review the code. I'll go through it tomorrow.

File: components/Icons/MobileIcons.js [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+        height="24"
+        role="img"
+        viewBox="0 0 24 24"
+        width="24"
+      >
+        <path
+          d="M19 10.5A8.5 8.5 0 1110.5 2a8.5 8.5 0 018.5 8.5z"
+          fill="none"
+          stroke="currentColor"
+          stroke-Linecap="round"
```
Comment (alshakero) [inline components/Icons/MobileIcons.js:112]:
```suggestion
          strokeLinecap="round"
```

File: components/NavBar/NavBar.module.css [hunk 2]
```diff
@@ -7,10 +7,31 @@
...
 }
+.mobileTopNavContainer {
+  width: 100%;
+  background-color: #fff;
+  border-bottom: 1px solid #dbdbdb;
+  position: fixed;
+  left: 0;
+  z-index: 2;
+  top: 0px;
+  height: 44px;
```
Comment (alshakero) [inline components/NavBar/NavBar.module.css:18]: I don't think you need to set the height. If you're not happy with how it looks, adjust paddings and margins.

```suggestion
```

File: components/NavBar/NavBar.module.css [hunk 3]
```diff
@@ -7,10 +7,31 @@
...
+}
+.mobileBottomNavContainer {
+  width: 100%;
+  background-color: #fff;
+  border-top: 1px solid #dbdbdb;
+  position: fixed;
+  left: 0;
+  z-index: 2;
+  top: 96%;
```
Comment (alshakero) [inline components/NavBar/NavBar.module.css:27]: Top 96% means the element height is 4%. Is it? 

You should do
```suggestion
  bottom: 0;
```

**Review by alshakero — CHANGES_REQUESTED** [review]
Great work. Left some notes.

**Comment by alshakerM** [comment]
Removed height and used padding 1px,
Edit on some code in  SVG to make it camelCase.
Remove 96% top and made it bottom 0

**Review by alshakero — APPROVED** [review]
LGTM. I'll merge this one because I need `useMediaQuery` in #100

**Pull request merged**